CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [v1.4.2] - 2024-01-09
```

Comment:
This link (and the one below for `v1.4.1` is broken, woops.

Should be `[1.4.2]`. Would you please fix that and the one below for `1.4.1`?